proxy-node-shared/src/test/java/uk/gov/ida/notification/contracts/verifyserviceprovider/AttributesBuilder.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.ida.notification.contracts.verifyserviceprovider;
+
+import org.joda.time.DateTime;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+
+public class AttributesBuilder {
+
+    public static String FIRST_NAME = "Jean";
+    public static String MIDDLE_NAME = "Paul";
+    public static String LAST_NAME = "Smith";
+    public static String GENDER = "NOT_SPECIFIED";
+    public static DateTime VALID_FROM = createDateTime(2001, 1, 1, 12, 0);
+    public static DateTime DATE_OF_BIRTH = createDateTime(1990, 1, 1, 0, 0);
+
+    private List<Attribute<String>> firstNames = new ArrayList<>(singletonList(createAttribute(FIRST_NAME)));
+    private Attribute<String> middleName = createAttribute(MIDDLE_NAME);
+    private Attribute<String> lastName = createAttribute(LAST_NAME);
+    private Attribute<String> gender = createAttribute(GENDER);
+    private Attribute<DateTime> dateOfBirth = createAttribute(DATE_OF_BIRTH);
+
+    public Attributes build() {
+        return new Attributes(
+                firstNames,
+                middleName != null ? singletonList(middleName) : emptyList(),
```

Comment:
Could this be a function?

Reply:
I don't think that's really needed - I'd leave it as it is